### ISSUE TITLE
Add missing attribute S3CanonicalUserId for AWS::CloudFront::CloudFrontOriginAccessIdentity

### ISSUE THREAD

**Issue by C-Otto** [shalupov/idea-cloudformation#131]
The `AWS::CloudFront::CloudFrontOriginAccessIdentity` resource provides an attribute named `S3CanonicalUserId `. Accessing it via `!GetAtt` results in an "unknown attribute" error.

My attempt at fixing it:

```
diff --git a/src/main/resources/cloudformation-metadata.xml b/src/main/resources/cloudformation-metadata.xml
index 1f271c3..cf32de5 100644
--- a/src/main/resources/cloudformation-metadata.xml
+++ b/src/main/resources/cloudformation-metadata.xml
@@ -3070,7 +3070,14 @@
             </ResourceProperty>
           </entry>
         </properties>
-        <attributes class="linked-hash-map"/>
+        <attributes class="linked-hash-map">
+          <entry>
+            <string>S3CanonicalUserId</string>
+            <ResourceAttribute>
+              <name>S3CanonicalUserId</name>
+            </ResourceAttribute>
+          </entry>
+        </attributes>
       </ResourceType>
     </entry>
     <entry>
```

**Comment by shalupov** [comment]
fixed in 0.5.68

**Comment by C-Otto** [comment]
Thank you! That was quick :)
